packages/components/src/styles/_utility.css

```diff
@@ -11,19 +11,28 @@
 
 .sd-toast-stack {
   position: fixed;
-  top: 0;
-  inset-inline-end: 0;
-  z-index: var(--sd-z-index-toast);
-  width: 28rem;
-  max-width: 100%;
+
+  z-index: 9999;
```

Comment:
Does Tailwind and css-nesting here not working? You could write the CSS classes as Tailwind Tokens with @apply.

Reply:
You're right. I added nesting on the bottom two styles and made them all consistent. And replaces the styles with @apply as well :)